nncf/experimental/torch/nas/bootstrapNAS/search/search.py

```diff
@@ -117,10 +128,16 @@ def from_dict(cls, search_config: Dict[str, Any]) -> 'SearchParams':
         mutation_eta = search_config.get('mutation_eta', 3.0)
         acc_delta = search_config.get('acc_delta', 1)
         ref_acc = search_config.get('ref_acc', -1)
+        aspiration_points = search_config.get('aspiration_points', [[0, 0]])
+        aspiration_points = np.array(aspiration_points)
+        epsilon = search_config.get('epsilon', 0.001)
+        weights = search_config.get('weights', None)
```

Comment:
Really minor comment, but I saw it in multiple places. The "get" method of dict returns "None" by default, no need to specify it. https://www.tutorialspoint.com/python/dictionary_get.htm

But maybe it's more a matter of taste, maybe it looks more explicit now